app/views/admin/competitors/index.html.erb

```diff
@@ -64,9 +64,9 @@
           <% end %>
         </td>
         <td>
-          <% if competitor.male? %>
+          <% if competitor.gender == 'male' %>
```

Comment:
I'd use `competitor.male?`/`competitor.female?` to hide the actual value representing male/female.